js/coinbase.js

```diff
@@ -683,31 +716,43 @@ module.exports = class coinbase extends Exchange {
         const ask = this.safeNumber (buy['data'], 'amount');
         const bid = this.safeNumber (sell['data'], 'amount');
         const last = this.safeNumber (spot['data'], 'amount');
-        return {
+        const ticker = {
             'symbol': symbol,
             'timestamp': timestamp,
-            'datetime': this.iso8601 (timestamp),
             'bid': bid,
             'ask': ask,
             'last': last,
+            'info': {
+                'buy': buy,
+                'sell': sell,
+                'spot': spot,
+            },
+        };
+        return this.parseTicker (ticker, market);
+    }
+
+    parseTicker (ticker, market = undefined) {
+        return {
```

Comment:
Let's always use `this.safeTicker` in return of all `parseTicker` methods (see how this is done in the other exchanges).